### PR TITLE
Dev 266

### PR DESCRIPTION
**Pull request by ratsagi**
I have included all missing alt tags and titles to logos on pages also in index.html. @mateahoward

### CODE REVIEW

**Review by J3ffJessie — APPROVED** [review]

**Comment by mateahoward** [comment]
Hey @ratsagi, we seem to have a conflict here, are you okay taking a look at that once more?

**Comment by mateahoward** [comment]
hey @J3ffJessie, don't forget to assign people to their tickets when reviewing so that we can track their contributions on dashboard as well as the reviewers 👍

File: index.html [hunk 1]
```diff
@@ -58,7 +58,7 @@ <h1>Junior Developer Group</h1>
         <a href="/pages/about.html">More Info</a>
       </div>
       <div class="hero_header-logo">
-        <img src="./assets/images/website-images/logo-with-background.png" />
+        <img src="./assets/images/website-images/logo-with-background.png"  alt="logo"/>
```
Comment (mateahoward) [inline index.html:61]: I'm not sure if you've used alt text before but 'logo' will not be an acceptable change here, unfortunately, have a look on discord - I have published instructions on how to write good alt text

File: pages/about.html [hunk 2]
```diff
@@ -43,7 +44,7 @@
     <section class="about-us-wrapper">
       <div class="about-us-content column">
         <div class="about-us__image">
-          <img src="/assets/images/website-images/logo-with-background.png" />
+          <img src="/assets/images/website-images/logo-with-background.png"  alt="logo"/>
```
Comment (mateahoward) [inline pages/about.html:47]: same as previous comment, we'll need to improve from just 'logo' to something more descriptive

File: pages/privacy-policy.html [hunk 3]
```diff
@@ -42,7 +44,7 @@
 
     <main>
       <section class="privacy-policy__header">
-        <img src="/assets/images/website-images/logo-with-background.png" />
+        <img src="/assets/images/website-images/logo-with-background.png" alt="logo-image" />
```
Comment (mateahoward) [inline pages/privacy-policy.html:47]: same as the other two comments, we'll need to add something a bit more descriptive

**Comment by mateahoward** [comment]
Perfect! Thank you for coming back to this @ratsagi - job well done 💯 🥇

**Review by mateahoward — APPROVED** [review]

**Pull request merged**